test/integration/js/user_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class UserTest < ActionDispatch::IntegrationTest
```

Comment:
I'd avoid calling this `UserTest` as that's also the name of the unit test and other integration test for the users controller.  Might as well put this in the existing users integration test perhaps?